com.trekglobal.idempiere.rest.api/src/com/trekglobal/idempiere/rest/api/json/NumericTypeConverter.java

```diff
@@ -67,7 +67,7 @@ public Object fromJsonValue(GridField field, JsonElement value) {
 	}
 	
 	private Object toJsonValue(int displayType, Number value) {
-		if (!DisplayType.isNumeric(displayType))
+		if (!(DisplayType.isNumeric(displayType) || displayType == Button))
```

Comment:
Hi @igorpojzl, all these lines show an error 

![grafik](https://user-images.githubusercontent.com/12065321/121886071-432d9480-cd15-11eb-93ce-dbd356976d10.png)

Maybe something wasn't added in the pull request?

Reply:
Commited missing Imports.